### PR TITLE
Delete concert command

### LINKED ISSUE #68: Delete concert command

### PR DESCRIPTION
**Pull request by wes-w-z-h**
- [x] new command to delete concerts
- [x] remove from storage
- [x] remove from concert contacts

Fixes #68

### CODE REVIEW

File: src/main/java/seedu/address/logic/parser/DeleteConcertCommandParser.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
+package seedu.address.logic.parser;
+
+import static seedu.address.logic.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.DeleteConcertCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+
+/**
+ * Parses input arguments and creates a new DeleteConcertComnnad object.
```
Comment (hj235) [inline src/main/java/seedu/address/logic/parser/DeleteConcertCommandParser.java:10]: typo: DeleteConcertComnnad

**Review by SKarthikeyan28 — COMMENTED** [review]
LGTM!

File: src/main/java/seedu/address/model/Model.java [hunk 2]
```diff
@@ -100,6 +100,11 @@ public interface Model {
      */
     void deleteConcertContact(Person targetPerson);
 
+    /**
+     * Deletes the given person. The person must exist in the address book.
```
Comment (hj235) [inline src/main/java/seedu/address/model/Model.java:104]: person -> concert

**Review by hj235 — COMMENTED** [review]
lgtm

**Review by hj235 — APPROVED** [review]
good job

**Pull request merged**